README.md

```diff
@@ -46,6 +46,33 @@ sudo podman run \
     quay.io/centos-bootc/fedora-bootc:eln
 ```
 
+### Using local containers
+
+To use containers from local container's storage rather than a registry, we need to ensure two things:
+- the container exists in local storage
+- mount the local container storage
+
+Since the container is run in `rootful` only root container storage paths are allowed.
+
+```bash
+sudo podman run \
+    --rm \
+    -it \
+    --privileged \
+    --pull=newer \
+    --security-opt label=type:unconfined_t \
+    -v $(pwd)/config.json:/config.json \
+    -v $(pwd)/output:/output \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    quay.io/centos-bootc/bootc-image-builder:latest \
+    --type qcow2 \
+    --config /config.json \
+    --local \
+    localhost/bootc:eln
+```
+
+Note we need to pass the `--local` flag, we need to mount the storage path as a volume and then use the name of the local container.
```

Comment:
Having an example use the term `localhost/` certainly emphasizes that a local container is supported...but we still support referring to non-`localhost/` images (e.g. `quay.io/...`) right?   So this sentence is misleading in saying "name of the local container" as if that is special in some way, as opposed to an image that happens to already be present in the container storage.

How about:

> When using the `--local` flag, we need to mount the storage path as a volume.  With this enabled, it is assumed that the target container is in the container storage.